spec/predicates/runtime-trace.md

```diff
@@ -105,40 +106,37 @@ Identifies the specific monitor instance used to trace the runtime.
 
 URI indicating the monitor’s identity.
 
-`process` _object_, _required_
-
-`process.HostID` _string (URI)_, _required_
+`monitor.configSource` _object_, _optional_
 
-URI indicating the process host’s identity. Ex: GitLab Runner ID
+Effectively a pointer to the monitor's configuration.
```

Comment:
Should this be a ResourceDescriptor type? The `entryPoint` could be specified as an `annotation`.

Reply:
Done! I've removed mention of `entryPoint` explicitly, I think that was a holdover from when this was based off SLSA Provenance.